noir-projects/noir-contracts/contracts/token_contract/src/main.nr

```diff
@@ -675,6 +675,7 @@ contract Token {
         fee_payer: AztecAddress, // Address of the entity which will receive the fee note.
         user: AztecAddress, // A user for which we are setting up the fee refund.
         funded_amount: Field, // The amount the user funded the fee payer with (represents fee limit).
+        nonce: Field, // A nonce to make authwitness unique.
```

Comment:
Added this as I was getting authwit nullifier collisions (the args were the same across multiple tests).